ragaai_catalyst/tracers/utils/rag_trace_json_converter.py

```
@@ -0,0 +1,197 @@
+    try:
+        for span in spans:
+            if span["name"] in ["ChatOpenAI", "ChatAnthropic", "ChatGoogleGenerativeAI"]:
+                start_time = datetime.fromisoformat(span.get("start_time", "")[:-1])  # Remove 'Z' and parse
+                end_time = datetime.fromisoformat(span.get("end_time", "")[:-1])    # Remove 'Z' and parse
+                additional_metadata["latency"] = (end_time - start_time).total_seconds()
+                additional_metadata["model_name"] = span["attributes"].get("llm.model_name", "").replace("models/", "")
+                additional_metadata["model"] = additional_metadata["model_name"]
+                try:
+                    additional_metadata["tokens"]["prompt"] = span["attributes"]["llm.token_count.prompt"]
+
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)